fixbackend/workspaces/router.py

```diff
@@ -179,11 +179,17 @@ async def delete_invite(
 
     @router.get("/{workspace_id}/accept_invite", name=ACCEPT_INVITE_ROUTE_NAME)
     async def accept_invitation(
-        token: str, invitation_service: InvitationServiceDependency, request: Request
+        token: str, workspace_id: WorkspaceId, invitation_service: InvitationServiceDependency, request: Request
     ) -> Response:
         """Accept an invitation to the workspace."""
         invitation = await invitation_service.accept_invitation(token)
-        url = request.base_url.replace_query_params(message="invitation-accepted", workspace_id=invitation.workspace_id)
+        if invitation is None:
```

Comment:
Better. 
It would be even better if we were idempotent and could say: "has been accepted already" or "Go away."
Is this hard to achieve?